src/tbb/tools_api/legacy/ittnotify.h

```diff
@@ -1,19 +1,8 @@
 /*
-    Copyright (c) 2005-2022 Intel Corporation
+  Copyright (C) 2005-2023 Intel Corporation
```

Comment:
Keep this line fixed but revert the license notice. We use Apache 2.0 for all ittnotify files.